lib/generators/react_on_rails/install_generator.rb

```diff
@@ -34,6 +34,13 @@ class InstallGenerator < Rails::Generators::Base
                    desc: "Install files necessary for deploying to Heroku",
                    aliases: "-H"
 
+      # --bootstrap
+      class_option :bootstrap,
+                   type: :boolean,
+                   default: false,
```

Comment:
@robwise let me make it default to `true`
@justin808 I will let this PR pass build today.